src/charts/axis.js

```diff
@@ -61,6 +61,8 @@ export default function Axis (_container) {
     verticalGridLines: null
   }
 
+  let xLabelsShouldRotate = false
```

Comment:
Why do you need a variable for that instead of calling `shouldXLabelsRotate` like in line 258? If you prefer to cache it like this, you could move it to the cache object
```
const cache = {
    container: _container,
    xLabelsShouldRotate: false,
```

Reply:
`cache` could be another place to store it. I'm trying to avoid calling `shouldXLabelsRotate()` multiple times when `axis.render()` is called.